### PR TITLE
#3 add githubRepo.json, closes #3

### LINKED ISSUE #3: Pull Request Demo

### PR DESCRIPTION
**Pull request by Predony**

### CODE REVIEW

File: mocks/githubRepo.json [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+      "html_url": "https://github.com/octocat",
+      "followers_url": "https://api.github.com/users/octocat/followers",
+      "following_url": "https://api.github.com/users/octocat/following{/other_user}",
+      "gists_url": "https://api.github.com/users/octocat/gists{/gist_id}",
+      "starred_url": "https://api.github.com/users/octocat/starred{/owner}{/repo}",
+      "subscriptions_url": "https://api.github.com/users/octocat/subscriptions",
+      "organizations_url": "https://api.github.com/users/octocat/orgs",
+      "repos_url": "https://api.github.com/users/octocat/repos",
+      "events_url": "https://api.github.com/users/octocat/events{/privacy}",
+      "received_events_url": "https://api.github.com/users/octocat/received_events",
```
Comment (ofKrown) [inline mocks/githubRepo.json:19]: Wie meinen?

**Review by ofKrown — COMMENTED** [review]
See comment

File: mocks/githubRepo.json [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+      "type": "User",
+      "site_admin": false
+    },
+    "name": "Hello-World",
+    "full_name": "octocat/Hello-World",
+    "description": "This your first repo!",
+    "private": false,
+    "fork": false,
+    "url": "https://api.github.com/repos/octocat/Hello-World",
+    "html_url": "https://github.com/octocat/Hello-World",
```
Comment (ofKrown) [inline mocks/githubRepo.json:29]: We should work on that

Comment (Predony) [reply mocks/githubRepo.json:29]: ok

**Review by ofKrown — APPROVED** [review]
Alles ok

**Pull request merged**